java/src/test/java/com/google/oak/crypto/EncryptorTest.java

```diff
@@ -17,90 +17,82 @@
 package com.google.oak.crypto;
 
 import com.google.oak.crypto.hpke.KeyPair;
-import com.google.oak.crypto.v1.AeadEncryptedMessage;
 import com.google.oak.crypto.v1.EncryptedRequest;
 import com.google.oak.crypto.v1.EncryptedResponse;
 import com.google.oak.util.Result;
-import com.google.protobuf.ByteString;
-import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-import java.util.Optional;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
 public class EncryptorTest {
-  private static final byte[] TEST_REQUEST_PLAINTEXT =
-      new byte[] {'R', 'e', 'q', 'u', 'e', 's', 't'};
-  private static final byte[] TEST_REQUEST_ASSOCIATED_DATA = new byte[] {'d', 'a', 't', 'a', '1'};
-  private static final byte[] TEST_RESPONSE_PLAINTEXT =
-      new byte[] {'R', 'e', 's', 'p', 'o', 'n', 's', 'e'};
-  private static final byte[] TEST_RESPONSE_ASSOCIATED_DATA = new byte[] {'d', 'a', 't', 'a', '2'};
+    private static final byte[] TEST_REQUEST_PLAINTEXT = new byte[] { 'R', 'e', 'q', 'u', 'e', 's', 't' };
```

Comment:
Could you confirm that formatting and its check works correctly in CI? It seems it currently doesn't, since this code was not consistently formatted.

Reply:
Yes, I noticed and fixed it now. Somehow VS Code has a per-file setting for indent spaces. Not sure who set it to 4, it is 2 now which greatly reduced the delta.

Reply:
Thanks, but can you also confirm that CI catches this as an error?

Reply:
No, it doesn't catch. I think this needs to be addressed better in general - how do we enforce that the Java written here conforms to the internal Java style as much as possible. Maybe not feasible to do this 100%, but a few large steps would help.

For example, the CI does not notice unused imports.

Should we have a bug for this? It affects C++ as well.